webpack/assets/javascripts/react_app/redux/API/APIReducer.js

```diff
@@ -14,8 +14,9 @@ const apiReducer = (state = initialState, { type, key, payload, response }) => {
     case REQUEST:
       return state.merge({
         [key]: {
-          payload,
           response: null,
+          ...state[key],
```

Comment:
For single API request there won't be any difference since the response is initially set to null as before,
though if the response already exist it will override the null value.